web/js/app.js

```diff
@@ -83,6 +84,15 @@ class App extends React.Component {
     keyPressAction(event.keyCode, event.shiftKey, ctrlOrCmdKey, isInput);
   }
 
+  getScreenInfo = () => {
+    const { setScreenInfoAction } = this.props;
```

Comment:
You could move all this code into the `setScreenInfo` action itself, so dispatching the action would just look like:

```
setScreenInfoAction: () => {
    dispatch(setScreenInfo());
  },
```

Reply:
Moved code into the setScreenInfo action.